integration

```diff
@@ -548,6 +548,7 @@
   "FaserF/ha-foodsharing",
   "FaserF/ha-kadermanager",
   "Favio25/Kronoterm-homeassistant",
+  "FeatherKing/ha-matriocontrol",
```

Comment:
https://github.com/FeatherKing/ha-matriocontrol/blob/1.0.5/hacs.json#L4 invalid filename, from what I can tell you should not define a filename at all.